### PR TITLE
Update the arm64 test drop - WIP

### PR DESCRIPTION
**Pull request by jashook**

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet test Windows_NT Arm64 Checked

**Comment by jashook** [comment]
@dotnet test Windows_NT Arm64 Checked

**Comment by jashook** [comment]
@dotnet test Windows_NT Arm64 Checked

**Comment by jashook** [comment]
@dotnet test Windows_NT Arm64 Checked

**Comment by jashook** [comment]
@dotnet test Windows_NT Arm64 Checked

**Comment by jashook** [comment]
@dotnet test Windows_NT Arm64 Checked

**Comment by jashook** [comment]
@dotnet test Windows_NT Arm64 Checked

**Comment by jashook** [comment]
@dotnet test Windows_NT Arm64 Checked

**Comment by danmosemsft** [comment]
@dotnet-bot test windows_nt arm64 cross checked build and test (looks like https://github.com/dotnet/coreclr/pull/10407)

**Comment by danmosemsft** [comment]
@jashook I do'nt know why that leg won't trigger.

**Comment by jashook** [comment]
@danmosemsft the arm64 jobs can only be triggered by a certain subset of people.

**Comment by danmosemsft** [comment]
@jashook who owns who is in that subset? Do you know the purpose of it? If we are all required to get the job to green, presumably we need rights to actually trigger it -- but maybe it's an optional job?
@mmitche

**Comment by mmitche** [comment]
@dotnet-bot test Windows_NT arm64 Checked

**Comment by mmitche** [comment]
@danmosemsft You may add yourself to the subset if you'd like (it's in the netci file)

**Comment by jashook** [comment]
Canceled the job, waiting for a rebuild of the tests before restarting.

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT arm64 Checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT arm64 Checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT arm64 Checked

**Comment by rahku** [comment]
@dotnet-bot test Windows_NT arm64 Checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT arm64 Checked

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT arm64 Checked

File: tests/scripts/lst_creator.py [hunk 1]
```diff
@@ -348,7 +351,14 @@ def main(args):
...
                         overwritten = True
-                        old_metadata[attribute] = ";".join(joined_categories)
+                        ordered_categories = []
+                        for item in old_split:
+                            if item in joined_categories:
+                                ordered_categories.append(item)
+                                joined_categories.remove(item)
+
+                        old_metadata[attribute] = ";".join(ordered_categories)
+                        old_metadata[attribute] = old_metadata[attribute] + ";" + ";".join(joined_categories) if len(joined_categories) > 0 else old_metadata[attribute]
```
Comment (RussKeldorph) [inline tests/scripts/lst_creator.py]: This line is a bit long and hard to parse, but not a big deal.
